src/components/logo.tsx

```diff
@@ -6,7 +8,7 @@ export const Logo = ({ isFooter }: Props) => (
         <svg className={`logo logo__white ${isFooter ? "logo--footer" : ""}`}>
             <use xlinkHref="#icon-logo-white" />
         </svg>
-        {!isFooter && (
+        {isFooter ? null : (
```

Comment:
[while I was there] I find expressions like `!isFooter && ` a bit hard to understand while we're browsing the code, so I converted it to a ternary :slightly_smiling_face: 
Also, using ternaries rather than `&&` expressions in JSX avoids some edge case bugs:
> https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx